raiden/network/blockchain_service.py

```diff
@@ -99,42 +98,14 @@ def __init__(
         self._payment_channel_creation_lock = Semaphore()
         self._user_deposit_creation_lock = Semaphore()
 
-    @property
-    def node_address(self) -> Address:
-        return self.client.address
-
-    def block_number(self) -> BlockNumber:
-        return self.client.block_number()
-
-    def block_hash(self) -> BlockHash:
```

Comment:
> But also semantically the block number /hash is also an attribute of the blockchain, no?

Sure, but as I wrote in https://github.com/raiden-network/raiden/issues/4690 the `BlockchainService` doesn't have much to do with the blockchain any more. It's mostly the place in which proxies get instantiated and stored.

This is just a first PR to get started as I had some time yesterday. Renaming the thing and replacing it with the client directly in some places will be next. Does that make sense?

Reply:
Then perhaps also rename the class to what you think it has to do with?

But what about the convenience argument I mentioned in https://github.com/raiden-network/raiden/pull/4965#discussion_r327477538 ?

Reply:
> Then perhaps also rename the class to what you think it has to do with?

I'd prefer to do that in a different PR

> But what about the convenience argument I mentioned in #4965 (comment) ?

I think this is just a symptom of the unclear concerns of both classes, so I'd leave it for now and them improve it once we rename the class and fix the usage. In other words, besides renaming we also should pass a `RPCClient` to classes that need to interact with the blockchain.

Reply:
> I'd prefer to do that in a different PR

I won't block the PR for this but I would see the renaming of the class as very tightly connected with removing the functions you removed. 

I see at the class now, with the name `BlockchainService` and wonder what that is.

And I would generally not want cosmetic changes like this one to be half-done because I am not sure when the next PR will be done and in the meantime we get more confusin codebase.

> In other words, besides renaming we also should pass a RPCClient to classes that need to interact with the blockchain.

That makes sense but I think this is not the time for cosmetic changes / refactorings like this.